README.md

```diff
@@ -54,87 +54,28 @@ Prerequisites
 
 1. Optional: libtock-c also includes support for building for RISC-V targets.
    These are not included by default since obtaining the toolchain can be
-   difficult (as of June 2020). You will need a `riscv64-unknown-elf` GCC
-   toolchain that supports rv32 targets as well (i.e. is compiled with multilib
-   support).
+   difficult (as of June 2020). You will need a toolchain that supports rv32
+   targets (i.e. is compiled with multilib support).
 
    To actually build for the RISC-V targets, add `RISCV=1` to the make command:
 
        $ make RISCV=1
 
    **MacOS**:
    ```
-   $ brew tap riscv/riscv && brew update && brew install riscv-gnu-toolchain --with-multilib
+   $ brew tap riscv/riscv && brew update && brew install riscv-gnu-toolchain
    ```
-   Warning: this will compile from source, and takes a while. Also this will
-   build a bleeding-edge version of GCC, and there is a small chance it will not
-   work. However, we have successfully obtained a toolchain this way.
 
    **Ubuntu (19.10 or later)**:
    ```
    $ sudo apt install gcc-riscv64-unknown-elf
    ```
 
-   Unfortunately, Ubuntu does not currently (June 2020) provide a package for
-   RISC-V libc. We have created a .deb file you can use to install a suitable
-   libc based on newlib:
-   ```
-   $ wget http://cs.virginia.edu/~bjc8c/archive/newlib_3.3.0-1_amd64.deb
-   $ sudo dpkg -i newlib_3.3.0-1_amd64.deb
-   ```
-
-   If you would rather compile your own newlib-based libc, follow the steps below.
-   Section [newlib-nano][newlib-nano] describes some extra config options to
-   build a size optimised newlib.
-   ```
-   # Download newlib 3.3 from https://sourceware.org/newlib/
-   wget ftp://sourceware.org/pub/newlib/newlib-3.3.0.tar.gz
-   tar -xvf newlib-3.3.0.tar.gz
-   cd newlib-3.3.0
-   # Disable stdlib for building
-   export CFLAGS=-nostdlib
-   # Run configure
-   ./configure --disable-newlib-supplied-syscalls --with-gnu-ld --with-newlib --enable-languages=c --target=riscv64-unknown-elf --host=x86 --disable-multi-lib --prefix /usr
-   # Build and then install
-   make -j8
-   sudo make install
-   ```
-
-   Alternatively, you may use a pre-compiled toolchain that we created with
-   Crosstool-NG.
-   ```
-   $ wget http://cs.virginia.edu/~bjc8c/archive/gcc-riscv64-unknown-elf-8.3.0-ubuntu.zip
-   $ unzip gcc-riscv64-unknown-elf-8.3.0-ubuntu.zip
-   # add gcc-riscv64-unknown-elf-8.3.0-ubuntu/bin to your `$PATH` variable.
-   ```
-
    **Arch**:
    ```
    $ sudo pacman -Syu riscv64-elf-gcc riscv32-elf-newlib
```

Comment:
It's worth removing the newlb dependency here

Reply:
At least on Ubuntu 20.04, I haven't gotten compiling `libtock-c` to work with the standard `gcc-riscv64-unknown-elf` package at it couldn't find some libs. I can reproduce this error if desired. Thus in my CI workflows I'm still resorting to the prebuilt package by @bradjc. If it doesn't work out of the box, we should leave these instructions in.

Reply:
That prebuilt package is just newlib, right? Since we have built packages in the repo now is it still required? Can we look at errors and maybe figure out what is going wrong?